### PR TITLE
Add: new feature

### PR DESCRIPTION
**Pull request by IndianMax03**
hello ksum what about this

### CODE REVIEW

**Comment by De-ad** [comment]
i will never forgive you.

File: index.html [hunk 1]
```diff
@@ -91,6 +91,11 @@
                     mineral: 'Алмаз',
                     workability: 'Царапает стекло',
                 },
+                {
+                    hardness: 10000,
```
Comment (De-ad) [inline index.html:95]: hardness: -1

**Comment by IndianMax03** [comment]
oh ksum thank u wery mach for yor opingeymer

File: index.html [hunk 2]
```diff
@@ -91,6 +91,11 @@
...
+                    mineral: 'Индус',
+                    workability: 'Царапает лицо',
```
Comment (De-ad) [inline index.html:97]: he has no nails

**Review by De-ad — COMMENTED** [review]
improve your skills.

**Pull request merged**